docs/whats-new/index.yml

```diff
@@ -0,0 +1,35 @@
+### YamlMime:Landing
+
+title: .NET Documentation What's New?
+summary: "Welcome to What's new in .NET docs. Quickly find the latest changes in .NET docs."
+
+metadata:
+  title: .NET Documentation What's New?
+  description: "Welcome to What's new in .NET docs."
+  ms.topic: landing-page
+  ms.date: 12/12/2019
+
+landingContent:
+  - title: "What's new"
+    linkLists:
+      - linkListType: whats-new
+        links:
+          - text: November 2019
+            url: 2019-11.md
+
+  - title: "Open source contributions"
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: ".NET Docs repositories overview."
+            url: https://github.com/dotnet/docs/blob/master/README.md
```

Comment:
This didn't change in November - what's the date range for this what's new list?

Reply:
For the landing page, I followed the [Xamarin example](https://docs.microsoft.com/xamarin/whats-new/).  They use that section to help people find ways to contribute. I'll change the titles to reflect that better.